lib/epicenter_web/live/contacts_live.ex

```diff
@@ -6,13 +6,14 @@ defmodule EpicenterWeb.ContactsFilter do
   alias Epicenter.AuditLog
 
   def render(assigns) do
-    ~H"""
+    ~M"""
     #status-filter
       = live_patch "All", to: Routes.contacts_path(@socket, EpicenterWeb.ContactsLive, filter: :with_contact_investigation), class: "button", data: [active: assigns.filter in [:with_contact_investigation, nil], role: "contacts-filter", tid: "all"]
       = live_patch "Pending interview", to: Routes.contacts_path(@socket, EpicenterWeb.ContactsLive, filter: :with_pending_interview), class: "button", data: [active: assigns.filter == :with_pending_interview, role: "contacts-filter", tid: "with_pending_interview"]
       = live_patch "Ongoing interview", to: Routes.contacts_path(@socket, EpicenterWeb.ContactsLive, filter: :with_ongoing_interview), class: "button", data: [active: assigns.filter == :with_ongoing_interview, role: "contacts-filter", tid: "with_ongoing_interview"]
       = live_patch "Quarantine monitoring", to: Routes.contacts_path(@socket, EpicenterWeb.ContactsLive, filter: :with_quarantine_monitoring), class: "button", data: [active: assigns.filter == :with_quarantine_monitoring, role: "contacts-filter", tid: "with_quarantine_monitoring"]
     """
+    |> Map.put(:root, true)
```

Comment:
I can investigate further, but I don't believe slim templates currently work with the `~H` sigil. I inspected the output of `~H` sigil, and it had `root: true` - so I added it to the output of compiled slim template.